Configuration/TCA/Overrides/tx_staffdirectory_domain_model_organization.php

```diff
@@ -77,17 +77,27 @@
         'exclude' => false,
         'label' => 'LLL:EXT:core/Resources/Private/Language/locallang_general.xlf:LGL.image',
         'l10n_mode' => 'exclude',
-        'config' => \TYPO3\CMS\Core\Utility\ExtensionManagementUtility::getFileFieldTCAConfig(
-            'tx_staffdirectoryorganization_images',
-            [
-                'maxitems' => 5,
-                'minitems' => 0
-            ],
-            $GLOBALS['TYPO3_CONF_VARS']['GFX']['imagefile_ext']
-        )
     ],
 ];
-
+$typo3Version = \TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance(\TYPO3\CMS\Core\Information\Typo3Version::class)->getMajorVersion();
+if ($typo3Version < 12) {
```

Comment:
Invert business logic, nowadays the most common case is v12 and up, thus "else" needs to come first.